product_dimension/migrations/12.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 C2i Change 2 improve - Eduardo Magdalena <emagdalena@c2i.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+field_renames = [
+    ('product.template.length', 'product.template.product_length', 
```

Comment:
invalid syntax
see : https://github.com/OCA/openupgradelib/blob/master/openupgradelib/openupgrade.py#L587

should be [
 ('product.template', 'product_template', 'length', 'product_length'),
 ('product.template', 'product_template', 'width', 'product_width'),
...